s2core/src/test/scala/com/kakao/s2graph/core/Integrate/IntegrateCommon.scala

```diff
@@ -113,48 +113,89 @@ trait IntegrateCommon extends FunSuite with Matchers with BeforeAndAfterAll {
     def getEdgesSync(queryJson: JsValue): JsValue = {
       logger.info(Json.prettyPrint(queryJson))
 
```

Comment:
Thanks!! Actually, I had the same thought just before I went to bed.
Made the changes.